src/pudl_archiver/archivers/eia861.py

```diff
@@ -22,21 +21,21 @@ async def get_resources(self) -> ArchiveAwaitable:
         link_pattern = re.compile(r"f861(\d{2,4}).zip")
 
         for link in await self.get_hyperlinks(BASE_URL, link_pattern):
-            yield self.get_year_resource(link, link_pattern.search(link))
-
-    async def get_year_resource(
-        self, link: str, match: typing.Match
-    ) -> tuple[Path, dict]:
+            year = int(link_pattern.search(link).group(1))
+            # Older file names only have last two digits of year in name
+            # Convert to 4-digit years
+            if year < 100 and year >= 90:
+                year += 1900
+            elif year < 100 and year < 90:
+                year += 2000
+                print(year)
```

Comment:
Do we need this print statement? Also a nitpick, we could just do `elif year < 90:`

Reply:
Nope, this was a debug statement that fell through the `git add -p` cracks. Good catch!